CHANGELOG.md

```diff
@@ -3,4 +3,6 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 1.2.0 Change how the gem is configured
```

Comment:
Literally don't care as it is in use by one person (me).  Once I publish to rubygems I'll care